src/app/artists/services/service.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
```

Comment:
creating service directory is redundant

Reply:
the name of service should be artist in singular
```ng g s artists/artist```

Reply:
listos los cambio